src/verifiers/revocationStore/openAttestationRevocationStore.test.ts

```diff
@@ -0,0 +1,75 @@
+import { openAttestationRevocationStore } from "./openAttestationRevocationStore";
+import { verificationBuilder } from "../verificationBuilder";
+import { documentRopstenValidWithDocumentStore } from "../../../test/fixtures/v2/documentRopstenValidWithDocumentStore";
+import { documentRopstenRevocationStoreNotRevoked, documentRopstenRevocationStoreRevoked } from "./revocationStore";
+
+const verify = verificationBuilder([openAttestationRevocationStore]);
+describe("OpenAttestationRevocationStore", () => {
+  describe("v2", () => {
+    it("should return a skipped fragment when document store is used", async () => {
+      const fragment = await verify(documentRopstenValidWithDocumentStore, { network: "ropsten" });
+      expect(fragment).toStrictEqual([
+        {
+          name: "OpenAttestationRevocationStore",
+          type: "DOCUMENT_STATUS",
+          reason: {
+            code: 4,
+            codeString: "SKIPPED",
+            message: 'Document issuers doesn\'t have "revocationStore"',
+          },
```

Comment:
were we giving reasons for skip for other fragments?

Reply:
yes for all